website/docs/components/form/textarea/partials/code/how-to-use.md

```diff
@@ -136,31 +136,29 @@ If you want to connect one or more extra elements describing the field to the co
 
 #### HTML native attributes
 
-As explained above in the [Component API](#component-api) section, the textarea "field" supports the `...attributes` spreading of HTML attributes over the `<textarea>` element. This means you can use all the standard HTML attributes of the `<textarea>` element.
+This component supports use of `...attributes`. This means you can use all the standard HTML attributes of the `<textarea>` element. This can be useful in case you want to add specific native behaviors to the field, that are not exposed directly by the component (e.g., providing a `name` for the control, or adding `min` `max` `minlength` `maxlength` `pattern` attributes to it).
```

Comment:
```suggestion
This component supports use of `...attributes`. This means you can use all the standard HTML attributes of the `<textarea>` element. This can be useful in case you want to add specific native behaviors to the field, that are not exposed directly by the component (e.g., providing a `name` for the control, or adding `min`, `max`, `minlength`, `maxlength`, or `pattern` attributes to it).
```